src/Twig.php

```diff
@@ -371,6 +371,11 @@ public function get_timber_filters()
                 },
             ],
 
+            // Numbers filters
+            'format_bytes' => [
```

Comment:
Can we use `size_format` instead of `format_bytes`? I feel like we don’t have to invent a new function name here.